sim/core/statweight.go

```diff
@@ -226,24 +220,26 @@ func CalcStatWeight(swr proto.StatWeightsRequest, statsToWeigh []stats.Stat, ref
 			statMod = meleeHitStatMod
 			if baseStats[stat] < melee2HHitCap && baseStats[stat]+statMod > melee2HHitCap {
 				panic("no")
```

Comment:
I left this debug panic in accidentally from changes yesterday... could you just take it out while you're here  xD

If this is the reason you commented the code, then uncomment it.

Reply:
You already changed it back, i'll just merge.